src/Microsoft.CodeQuality.Analyzers.Exp/Core/FlowAnalysis/Analysis/NullAnalysis/NullAnalysis.NullAbstractValueDomain.cs

```diff
@@ -0,0 +1,57 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+
+namespace Microsoft.CodeAnalysis.Operations.DataFlow.NullAnalysis
+{
+    using NullAnalysisData = IDictionary<ISymbol, NullAbstractValue>;
```

Comment:
You have this in 4 different files. Why not just create a marker interface like

```cs
interface INullAnalysisData : IDictionary<ISymbol, NullAbstractValue> {}
```

?

Reply:
Its just a preference to avoid unncessary marker interfaces. We can change this later if we feel otherwise.

---
In reply to: [162773087](https://github.com/dotnet/roslyn-analyzers/pull/1541#discussion_r162773087) [](ancestors = 162773087)